docs/pages/guides/authentication/nuxt-js.mdx

```diff
@@ -70,8 +70,9 @@ app.post("/auth", (req, res) => {
     userId: "123", // Optional
     groupIds: ["456"], // Optional
     userInfo: {
-      // Optional
+      // Optional, corresponds to the UserMeta.info type defined in liveblocks.config.ts
```

Comment:
Same: It's not the same text that we use in other similar places, we should harmonize